static/app/types/type-loader.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '!!type-loader!*' {
```

Comment:
I would have preferred to colocate this with loader implementation, but it doesn't seem like our ts config will only recognize declaration files inside this dir and since we have the fileLoader defined here already, I finally opted to place it here